### DISCUSSION TITLE
主线程是否可以执行PAGFile.Load

### DISCUSSION THREAD

**Discussion by iGao101** [Tencent/libpag#1981] [Q&A] [answered]
PAGFile.setPath中调用了PAGFile.Load加载pag文件，给的demo中也是在主线程中加载的pag文件，但是如果pag文件比较大也是可以放到主线程的吗？想确认一下

**Comment by kevingpqi123** [comment, marked as answer]
PAGFile.Load 方法的执行不区分线程，放在子线程是可以的哈，在 PAG 4.3 版本中，我们增加了异步加载的接口，具体可以看下 API 文档

**Reply by iGao101** [reply]
好的，所以加载pag文件的过程也是耗时操作，对于比较大的pag文件还是建议使用异步接口进行加载么？
